wagtailquickcreate/views.py

```diff
@@ -25,7 +25,8 @@ def get_context_data(self, **kwargs):
         _model = kwargs.pop('model')
         app = kwargs.pop('app')
         model = apps.get_model(app, _model)
-        parent_models = model.allowed_parent_page_models()
+        # Exclude base wagtail page class from possible parents
+        parent_models = [m for m in model.allowed_parent_page_models() if m is not Page]
```

Comment:
```suggestion
        parent_models = [m for m in model.allowed_parent_page_models() if not isinstance(m, Page)]
```

Reply:
Looks like I may have been hasty with this 🤦

Reply:
No worries, I learnt not to just blindly accept code review suggestions, even from the best developers :)